### PR TITLE
feat(27): passion feed

### PR DESCRIPTION
**Pull request by KngZhi**
- feat: init passion-feed model
- feat: resolver for passion feed

#27 
```graphql
query MyQuery {
  passionFeed(account: "Fksmad33PFxhrQXNYPPJozgWrv82zuFLvXK7Rh8m1xQhe98") {
    id
  }
}
```

image

### CODE REVIEW

File: src/server-extension/resolvers/passionFeed.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import { Arg, Field, ObjectType, Query, Resolver, FieldResolver, Root } from 'type-graphql'
+import type { EntityManager } from 'typeorm'
+import { NFTEntity } from '../../model/generated'
+import { PassionFeedEntity } from '../model/passion.model'
+
+
+const passionQuery = `SELECT DISTINCT ne.issuer as id
```
Comment (vikiival) [inline src/server-extension/resolvers/passionFeed.ts:7]: move to `src/server-extension/query/nft.ts`

File: src/server-extension/resolvers/passionFeed.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+@Resolver()
+export class PassionFeedResolver {
+    constructor(private tx: () => Promise<EntityManager>) { }
+
+    @Query(() => [PassionFeedEntity])
+    async passionFeed(
+        @Arg('account', { nullable: false, }) account: string
+    ): Promise<[PassionFeedEntity]> {
+        const manager = await this.tx()
```
Comment (vikiival) [inline src/server-extension/resolvers/passionFeed.ts:20]: Please use `makeQuery` from `src/server-extension/utils.ts`

**Review by vikiival — CHANGES_REQUESTED** [review]
Smol stuff

**Comment by vikiival** [comment]
Really nice job @KngZhi !

**Comment by KngZhi** [comment]
> Really nice job @KngZhi !

Thanks for your encouragement! 😄 and 🎉

**Review by vikiival — APPROVED** [review]

**Pull request merged**